frontend/src/components/UI/TabPanels/Tabs.vue

```diff
@@ -157,6 +159,130 @@
 						</template>
 					</Modal>
 				</q-tab-panel>
+
+				<q-tab-panel name="childs" class="flex q-gutter-md">
+					<div
+						class="cards_wrapper"
+						v-for="child in workspaceChilds"
+						:key="child.ID"
+					>
+						<div v-if="child.Logo">
```

Comment:
Do not make a if only for one attribut.
Please: 
```
:logo="child.Logo || logo" // or something else
```

Reply:
Ok, I'ill do this : :logo="child.Logo ? child.Logo : logo"